app/controllers/curation_concerns/generic_works_controller.rb

```diff
@@ -9,6 +9,8 @@ class CurationConcerns::GenericWorksController < ApplicationController
   self.show_presenter = AssetPresenter
   skip_load_and_authorize_resource only: :manifest
 
+  after_action :set_uploaded_file_status, only: [:create]
```

Comment:
oh nice, so when individual assets are created by imaging, they will also set the uploaded files to "begun_ingestion"...